spec/apis/navigate-to-url.spec.js

```diff
@@ -66,13 +66,11 @@ export default function() {
 			expectPathAndHashToEqual('somethinger#b/my-route');
 		});
 
-		it(`should call push state when the origin's don't match, since single spa never wants to refresh the page`, function() {
```

Comment:
This test was testing exactly what it said it was. But the behavior was just wrong haha. You can't call pushState on a url that is a different domain than the one you are currently on.

Reply:
😀